sdk/eventgrid/Azure.Messaging.EventGrid/CHANGELOG.md

```diff
@@ -1,15 +1,27 @@
 # Release History
 
-## 4.17.0-beta.1 (Unreleased)
+## 4.17.0 (2023-06-08)
 
 ### Features Added
 
+- Added new system events for Container Service. 
+
 ### Breaking Changes
 
-### Bugs Fixed
+- `EventGridClient` is removed from the beta releases. It will be included in a subsequent release.
```

Comment:
```suggestion
- `EventGridClient` had been removed for the stable release.  It will be included in the next beta release.
```

Reply:
I'm not sure if we will have another beta for it or not.

Reply:
Feel free to tune.   The original text read to me like we were taking `EventGridClient` out of a beta release - but this is a stable one.

Reply:
Ah yeah - you are right. Fixed.